src/js/pensions/containers/ConfirmationPage.jsx

```diff
@@ -54,8 +55,18 @@ class ConfirmationPage extends React.Component {
               <strong>Date received</strong><br/>
               <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
             </li>
+            <li>
+              <strong>Confirmation number</strong><br/>
+              <span>{response.confirmationNumber}</span>
+            </li>
+            <li>
+              <strong>Your claim was sent to</strong><br/>
```

Comment:
I think that's in the upload ticket you were working on yesterday, to add a message about that here.